test/integration/test_spark_lightning.py

```diff
@@ -907,71 +811,6 @@ def test_train_with_inmemory_cache_all(self):
                 assert len(pred) == 1
                 assert pred.dtype == torch.float32
 
-    """
-    Test pytorch lightning trainer with origin petastrom reader.
-    """
-    def test_origin_petastorm_reader(self):
```

Comment:
Are we removing the support of origin petastorm reader?

Reply:
Why keep things specific to Petastrom? Users don't need to know that.